src/eth/Web3Service.js

```diff
@@ -92,25 +92,23 @@ export default class Web3Service extends PrivateService {
   async connect() {
     this.get('log').info('Web3 is connecting...');
 
-    this._info.version = await promiseProps({
+    this._info = await promiseProps({
       api: this._web3.version,
       node: promisify(this._web3.eth.getNodeInfo)(),
       network: promisify(this._web3.eth.net.getId)(),
       ethereum: promisify(this._web3.eth.getProtocolVersion)()
     });
 
-    if (!this._info.version.node.includes('MetaMask')) {
-      this._info.version.whisper = await promisify(
-        this._web3.version.getWhisper
-      )().catch(() => '');
+    if (!this._info.node.includes('MetaMask')) {
+      this._info.whisper = this._web3.shh;
```

Comment:
this could be removed as I don't think whisper is used anywhere. I changed it to add the up to date web3.shh object all the same